lib/string.cpp

```diff
@@ -23,6 +23,7 @@
 #endif
 
 #include <cstring>
+#include <iostream>
```

Comment:
OK, the code references the standard streams directly.  Will fix this together with the other one and attribute this to you.